numpy/core/tests/test_mem_policy.py

```diff
@@ -89,6 +89,7 @@ def get_module(tmp_path):
          """),
     ]
     prologue = '''
+        #define NPY_TARGET_VERSION NPY_1_22_API_VERSION
         #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
         #include <numpy/arrayobject.h>
```

Comment:
@mattip this is also a nice confirmation.  Without this define build fails for the memory policy tests, since it requires API newer than 1.17.